apps/ledger-live-desktop/src/renderer/families/celo/AccountHeaderManageActions/AccountHeaderManageActions.jsx

```diff
@@ -8,34 +8,49 @@ import { accountsSelector } from "~/renderer/reducers/accounts";
 import { openModal } from "~/renderer/actions/modals";
 import Icon from "./Icon";
 import type { Account } from "@ledgerhq/types-live";
+import { isAccountEmpty } from "@ledgerhq/live-common/account/index";
 
 type Props = {
   account: Account,
+  parentAccount: ?Account,
 };
 
-const AccountHeaderManageActions = ({ account }: Props) => {
+const AccountHeaderManageActions = ({ account, parentAccount }: Props) => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const accounts = useSelector(accountsSelector);
+
   const isRegistrationPending = isAccountRegistrationPending(account.id, accounts);
 
   const onClick = useCallback(() => {
-    dispatch(
-      openModal("MODAL_CELO_MANAGE", {
-        account,
-      }),
-    );
-  }, [dispatch, account]);
+    if (isAccountEmpty(account)) {
+      dispatch(
+        openModal("MODAL_NO_FUNDS_STAKE", {
+          account,
+          parentAccount,
+        }),
+      );
+    } else {
+      dispatch(
+        openModal("MODAL_CELO_MANAGE", {
+          account,
+        }),
+      );
+    }
+  }, [account, dispatch, parentAccount]);
+
+  const disabledLabel = isRegistrationPending
```

Comment:
@sarneijim I assumed we should display `titleWhenPendingRegistration` when `isRegistrationPending` - can you double check if this is right?

Reply:
Good catch!